modules/aerodyn/src/UnsteadyAero.f90

```diff
@@ -2723,37 +2748,47 @@ subroutine UA_CalcContStateDeriv( i, j, t, u_in, p, x, OtherState, AFInfo, m, dx
       dxdt%x(3) = 0.0_R8Ki
       dxdt%x(5) = 0.0_R8Ki
 
-   elseif (p%UAMod == UA_HGMV) then
+   elseif (p%UAMod == UA_HGMV .OR. p%UAMod == UA_HGMV360) then
 
       call AFI_ComputeAirfoilCoefs( alphaE, u%Re, u%UserProp, AFInfo, AFI_AlphaE, ErrStat2, ErrMsg2)
          call SetErrStat(ErrStat2,ErrMsg2,ErrStat,ErrMsg,RoutineName)
          if (ErrStat >= AbortErrLev) return
 
-      Clp = AFI_AlphaE%FullyAttached + pi * Tu * u%omega                                       ! Eq. 13 (this is really Cnp)
-      dxdt%x(3) = -1.0_R8Ki / BL_p%T_p                                  * x%x(3) +         1.0_ReKi / BL_p%T_p  * Clp
-         
-      dxdt%x(4) = -1.0_R8Ki / BL_p%T_f0                                 *    x4  +         1.0_ReKi / BL_p%T_f0 * AFI_AlphaF%f_st
+      TuOmega = Tu * u%omega
+      TuOmega = MIN( MAX(TuOmega, -MaxTuOmega), MaxTuOmega)
+
+      Clp = AFI_AlphaE%FullyAttached + pi * TuOmega                                        ! Eq. 13 (this is really Cnp)
+      dxdt%x(3) = ( Clp             - x%x(3) ) / BL_p%T_p
+      dxdt%x(4) = ( AFI_AlphaF%f_st - x4     ) / BL_p%T_f0
```

Comment:
Can these modified lines be copied to the UA_HGM as well?

Reply:
Done. Good catch, @ebranlard.